### ISSUE TITLE
Custom Polus toggles don't get their assigned icon

### ISSUE THREAD

**Issue by Aransh** [ioscreatix/Maize#102]
The correct icon appears in Maize's setting, but on the CC itself it's blank. Also, the toggled required a respring to even appear...

**Comment by ahanank** [comment]
Encountered this bug too.

**Comment by andrewwiik** [comment]
Damn Polus and how it loads it's icons :/

**Comment by ahanank** [comment]
Wouldn't it be feasible if Maize had its own customization section for custom icons, eliminating Polus altogether? It already has arrangement under control.

**Comment by andrewwiik** [comment]
No, because Polus own the copy right to all those icon designs it has bundled, I cannot just plop them into Maize

**Comment by ahanank** [comment]
The icon designs can be custom made, as is the case with Polus.
